content/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file.md

```diff
@@ -15,7 +15,7 @@ shortTitle: Community health file
 
 ## About default community health files
 
-You can add default community health files to the root of a public repository called `.github` that is owned by an organization{% ifversion fpt or ghae or ghes %} or user account{% endif %}.
+You can add default community health files to a public repository called `.github`, that is owned by an organization or user account. You can put the files in the root of the repository or in the `docs` or `.github` folders.
```

Comment:
On second thought... all repositories are owned by organizations or user accounts, so this is no longer necessary.

```suggestion
You can add default community health files to a public repository called `.github`, in the root of the repository or in the `docs` or `.github` folders.
```